Bio/Restriction/Restriction.py

```diff
@@ -232,6 +238,18 @@ def finditer(self, pattern, size):
         return [(i.start(), i.group) for i in re.finditer(pattern, data)]
 
     def __getitem__(self, i):
+        """Return substring of ``FormattedSeq``.
+
+        The class of the returned object is the class of the respective
+        sequence. Note that due to the leading space, indexing is 1-based::
```

Comment:
You've got a double colon meaning an indented literal block should follow - but its a doctest. This explains the flake8 error:

```
flake8
./Bio/Restriction/Restriction.py:250:1: RST399 Inconsistent literal block quoting.
ERROR: InvocationError for command /home/travis/build/biopython/biopython/.tox/style/bin/flake8 (exited with code 1)
```

Just remove the extra colon.

Reply:
Yes.